app/assets/images/payment_icons/postfinance_card.svg

```diff
@@ -0,0 +1 @@
></linearGradient><path d="M7.7 10.6v2.1c0 .1-.1.2-.2.2H5.9c-.1 0-.2-.1-.2-.2V9.2c0-.2-.2-.4-.4-.4h-2c.1-.5.5-1 1.1-1h1.2v.8c-.1.2.1.4.3.4h4c.1 0 .2.1.2.2v.8c0 .1-.1.2-.2.2H8.1c-.2 0-.4.2-.4.4z" fill="url(#pi-SVGID8_)"/><linearGradient id="pi-SVGID9_" gradientUnits="userSpaceOnUse" x1="-225.046" y1="-147.024" x2="-219.296" y2="-151.983" gradientTransform="translate(229.79 161.584)"><stop offset="0" stop-color="#fbcc47"/><stop offset=".2" stop-color="#d19737"/><stop offset=".28" stop-color="#9c7533"/><stop offset=".53" stop-color="#dab23a"/><stop offset=".611" stop-color="#ca9f39"/><stop offset=".76" stop-color="#aa7735"/><stop offset=".806" stop-color="#af7d37"/><stop offset=".87" stop-color="#be8d3a"/><stop offset=".943" stop-color="#daaa40"/><stop offset="1" stop-color="#fbcc47"/></linearGradient><path d="M10.1 10.4v.8c0 .1-.1.2-.2.2H8.1c-.1 0-.2-.1-.2-.2v-.6c0-.1.1-.2.2-.2h2z" fill="url(#pi-SVGID9_)"/><linearGradient id="pi-SVGID10_" gradientUnits="userSpaceOnUse" x1="-224.39" y1="-146.263" x2="-218.64" y2="-151.223" gradientTransform="translate(229.79 161.584)"><stop offset="0" stop-color="#fbcc47"/><stop offset=".2" stop-color="#d19737"/><stop offset=".28" stop-color="#9c7533"/><stop offset=".53" stop-color="#dab23a"/><stop offset=".611" stop-color="#ca9f39"/><stop offset=".76" stop-color="#aa7735"/><stop offset=".806" stop-color="#af7d37"/><stop offset=".87" stop-color="#be8d3a"/><stop offset=".943" stop-color="#daaa40"/><stop offset="1" stop-color="#fbcc47"/></linearGradient><path d="M7.8 11.8c0-.1.1-.2.2-.2h1.8c.1 0 .2.1.2.2v.8c0 .1-.1.2-.2.2H8.1c-.1 0-.2-.1-.2-.2v-.8z" fill="url(#pi-SVGID10_)"/><linearGradient id="pi-SVGID11_" gradientUnits="userSpaceOnUse" x1="-223.882" y1="-145.675" x2="-218.133" y2="-150.633" gradientTransform="translate(229.79 161.584)"><stop offset="0" stop-color="#fbcc47"/><stop offset=".2" stop-color="#d19737"/><stop offset=".28" stop-color="#9c7533"/><stop offset=".53" stop-color="#dab23a"/><stop offset=".611" stop-color="#ca9f39"/><stop offset=".76" stop-color="#aa7735"/><stop offset=".806" stop-color="#af7d37"/><stop offset=".87" stop-color="#be8d3a"/><stop offset=".943" stop-color="#daaa40"/><stop offset="1" stop-color="#fbcc47"/></linearGradient><path d="M9 14H7.8v-.7c0-.1.1-.2.2-.2h2c0 .5-.4.9-1 .9z" fill="url(#pi-SVGID11_)"/><linearGradient id="pi-SVGID12_" gradientUnits="userSpaceOnUse" x1="-226.782" y1="-149.038" x2="-221.033" y2="-153.997" gradientTransform="translate(229.79 161.584)"><stop offset="0" stop-color="#fbcc47"/><stop offset=".2" stop-color="#d19737"/><stop offset=".28" stop-color="#9c7533"/><stop offset=".53" stop-color="#dab23a"/><stop offset=".611" stop-color="#ca9f39"/><stop offset=".76" stop-color="#aa7735"/><stop offset=".806" stop-color="#af7d37"/><stop offset=".87" stop-color="#be8d3a"/><stop offset=".943" stop-color="#daaa40"/><stop offset="1" stop-color="#fbcc47"/></linearGradient><path d="M5.9 8.8c-.1 0-.2-.1-.2-.2v-.7H9c.6 0 1 .4 1.1 1l-4.2-.1z" fill="url(#pi-SVGID12_)"/></g></svg>
```

Comment:
The same comments apply here.

Reply:
refactored the styles to attributes and adjusted borders, see: 28efe2c997fe2b35e49aec75a5644ea39c39d2cf